server/src/routers/api/userRouter.js

```diff
@@ -41,4 +48,15 @@ router.get("/", auth, (req, res) => {
   res.send(req.user);
 });
 
+// @route POST api/users/logout
+// @desc Logout a user
+// @access Private
+
+router.post("/logout", auth, async (req, res) => {
```

Comment:
If you can make the below listed changes by today, I will merge the PR and you will be awarded points else not.

logged in user is available at req.user and the token he has been using to access the private routes is available  at req.token. Why? All of this has been handled in the auth middleware. 
Also all the tokens he has been given so far is present in tokens array, (req.user.tokens)

1. To log him out, simply delete req.token from req.user.tokens.

Also, You should know how to test your API endpoints.
https://www.youtube.com/watch?v=t5n07Ybz7yI
Here's a video that might be helpful.